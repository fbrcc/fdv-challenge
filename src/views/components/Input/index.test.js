import React from 'react';
import { Input } from './index'
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
      .create(<Input  
        placeholder="Nombre" 
        name="name" 
        type="text" 
        value={"FermÃ­n Breccia"}
            >
        </Input>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });