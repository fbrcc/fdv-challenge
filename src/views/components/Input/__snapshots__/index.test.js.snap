// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<input
  className="sc-bdVaJa jWUjVV"
  name="name"
  onChange={[Function]}
  pattern={undefined}
  placeholder="Nombre"
  type="text"
  value="FermÃ­n Breccia"
/>
`;
